### PR TITLE
Constant Configurable Block Rewards by Admin 

### PR DESCRIPTION
**Pull request by abhishekvispute**
Hey Hi,
This PR 
Fixes https://github.com/razor-network/contracts/issues/74.

As per the issue, a constant block reward is needed which admin can update anytime.
We achieved so by constructor and update defined in StakeManager.
The update is restricted to admin only by ACL.
Removed calculation of block reward as per the curve.

Thanks

### CODE REVIEW

File: test/helpers/constants.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const EPOCH_LENGTH = BigNumber.from(40);
 const NUM_BLOCKS = 10;
 const NUM_STATES = BigNumber.from(4);
 const STATE_LENGTH = BigNumber.from(10);
+const BLOCK_REWARD = BigNumber.from(40).mul((BigNumber.from(10).pow(BigNumber.from(18))));
```
Comment (dev1644) [inline test/helpers/constants.js:10]: > BigNumber.from(40).mul((BigNumber.from(10).pow(BigNumber.from(18))));

Can be re-written as `BigNumber.from(40).mul(ONE_ETHER);`

File: test/ACL.js [hunk 2]
```diff
@@ -245,6 +246,11 @@ describe('Access Control Test', async () => {
     await stakeManager.revokeRole(await constants.getStakerActivityUpdaterHash(), signers[0].address);
   });
 
+  it('Only Default Admin should able to update Block Reward', async () => {
+    await assertRevert(stakeManager.connect(signers[1]).updateBlockReward(100), expectedRevertMessage);
+    assert(await stakeManager.updateBlockReward(100), "Admin not able to update BlockReward");
```
Comment (dev1644) [inline test/ACL.js:251]: Let's add one more case, where it `updateBlockReward` will be called by `non-admin` account & it should revert. @abhishekvispute

Comment (abhishekvispute) [reply test/ACL.js:251]: Yes, that's what line no. 250 does @dev1644

File: test/helpers/constants.js (hunk 1 above)
Comment (abhishekvispute) [reply test/helpers/constants.js]: Done, Thanks

File: test/ACL.js (hunk 2 above)
Comment (dev1644) [reply test/ACL.js:251]: Ok, Sounds good, I overlooked it.

**Review by dev1644 — APPROVED** [review]

**Review by SamAg19 — APPROVED** [review]

**Review by SkandaBhat — APPROVED** [review]

**Pull request merged**